app/assets/javascripts/live-site-search.js

```diff
@@ -13,157 +13,159 @@
+        liveSearch.$ariaLiveResultCount = $('.js-aria-live-count')
+        if (liveSearch.$form) {
+          liveSearch.$resultsBlock = $('.js-live-search-results-block')
 
-          liveSearch.action = liveSearch.$form.attr('action') + '.json';
+          liveSearch.action = liveSearch.$form.attr('action') + '.json'
 
-          liveSearch.saveState();
+          liveSearch.saveState()
 
-          liveSearch.$form.on('change', 'input[type=checkbox]', liveSearch.checkboxChange);
-          $(window).on('popstate', liveSearch.popState);
+          liveSearch.$form.on('change', 'input[type=checkbox]', liveSearch.checkboxChange)
+          $(window).on('popstate', liveSearch.popState)
         }
       } else {
-        $('.js-live-search-fallback').show();
+        $('.js-live-search-fallback').show()
       }
     },
-    popState: function(event){
-      if(event.originalEvent.state){
-        liveSearch.saveState(state);
-        liveSearch.updateResults();
-        liveSearch.restoreCheckboxes();
-        liveSearch.pageTrack();
+    popState: function (event) {
+      if (event.originalEvent.state) {
+        liveSearch.saveState(event.originalEvent.state)
+        liveSearch.updateResults()
+        liveSearch.restoreCheckboxes()
+        liveSearch.pageTrack()
       }
     },
-    pageTrack: function(){
+    pageTrack: function () {
       GOVUK.analytics.trackPageview(
         window.location.pathname + window.location.search,
         null,
         {
           dimension5: String(liveSearch.cache().result_count)
         }
-      );
-      $(document).trigger("liveSearch.pageTrack");
-    },
-    checkboxChange: function(e){
-      var pageUpdated;
-      if(liveSearch.checkFilterLimit(e) && liveSearch.isNewState()){
-        liveSearch.saveState();
-        pageUpdated = liveSearch.updateResults();
-        pageUpdated.done(function(){
-          history.pushState(liveSearch.state, '', window.location.pathname + "?" + $.param(liveSearch.state));
-          liveSearch.pageTrack();
-        });
+      )
+      $(document).trigger('liveSearch.pageTrack')
+    },
+    checkboxChange: function (e) {
+      var pageUpdated
+      if (liveSearch.checkFilterLimit(e) && liveSearch.isNewState()) {
+        liveSearch.saveState()
+        pageUpdated = liveSearch.updateResults()
+        pageUpdated.done(function () {
+          window.history.pushState(liveSearch.state, '', window.location.pathname + '?' + $.param(liveSearch.state))
+          liveSearch.pageTrack()
+        })
       }
     },
-    checkFilterLimit: function(e){
-      var newState = liveSearch.$form.serializeArray(),
-          filterCount = 0,
-          i, _i;
+    checkFilterLimit: function (e) {
+      var newState = liveSearch.$form.serializeArray()
 
-      for(i=0,_i=newState.length; i<_i; i++){
-        if(newState[i].name.lastIndexOf('filter_', 0) === 0){
-          filterCount = filterCount + 1;
+      var filterCount = 0
+
```

Comment:
Nitpick: probably don't need a newline between every variable declaration.

Reply:
StandardJS does that for us 🤷‍♂ think we'd better leave it. Removing the newlines would be us imposing our own nitpicks on the standards.

Reply:
Oh really? Oh well.

Reply:
Removing the newlines and then running standard with or without `--fix` doesn't flag or fix the change. (Double-checked it was actually running by adding a semicolon.) It seems that this doesn't seem to matter either way.